src/components/alert/alert.tsx

```diff
@@ -299,14 +296,25 @@ export class Alert implements OpenCloseComponent {
   /* @internal */
   @State() requestedIcon?: string;
 
-  private activeTransitionProp = "opacity";
+  activeTransitionProp = "opacity";
+
+  transitionEl: HTMLDivElement;
+
+  private transitionStartHandler = transitionStartHandler.bind(this);
```

Comment:
@Elijbet can we pass the component instead? That way its consistent with the other utils and we don't have to bind these functions in every component.